### PR TITLE
Fixing issue with pppp lan-search

### LINKED ISSUE #95: [BUG] `pppp lan-search` broken in upcoming

### PR DESCRIPTION
**Pull request by billyjbryant**
#### Description

Since `upcoming/v1.0.0`, `pppp lan-search` has failed to function with the following error: 

```sh
Traceback (most recent call last):
  File "/mnt/c/Users/billy/Projects/ankermake-m5-protocol/./ankerctl.py", line 488, in <module>
    main()
  File "/home/billy/.local/lib/python3.10/site-packages/click/core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
  File "/home/billy/.local/lib/python3.10/site-packages/click/core.py", line 1055, in main
    rv = self.invoke(ctx)
  File "/home/billy/.local/lib/python3.10/site-packages/click/core.py", line 1657, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/billy/.local/lib/python3.10/site-packages/click/core.py", line 1657, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/billy/.local/lib/python3.10/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/billy/.local/lib/python3.10/site-packages/click/core.py", line 760, in invoke
    return __callback(*args, **kwargs)
  File "/home/billy/.local/lib/python3.10/site-packages/click/decorators.py", line 84, in new_func
    return ctx.invoke(f, obj, *args, **kwargs)
  File "/home/billy/.local/lib/python3.10/site-packages/click/core.py", line 760, in invoke
    return __callback(*args, **kwargs)
  File "/mnt/c/Users/billy/Projects/ankermake-m5-protocol/./ankerctl.py", line 227, in pppp_lan_search
    api.send(PktLanSearch())
  File "/mnt/c/Users/billy/Projects/ankermake-m5-protocol/libflagship/ppppapi.py", line 359, in send
    raise ConnectionError(f"Tried to send packet in state {self.state}")
ConnectionError: Tried to send packet in state PPPPState.Idle
[*] ServiceManager: Shutdown complete
```

This PR fixes this issue. 

fixes #95

### CODE REVIEW

File: ankerctl.py [hunk 1]
```diff
@@ -224,6 +224,7 @@ def pppp_lan_search(env):
     """
     api = cli.pppp.pppp_open_broadcast(dumpfile=env.pppp_dump)
     try:
+        api.state = PPPPState.Connecting
```
Comment (chrivers) [inline ankerctl.py:227]: This change should go in `pppp_open_broadcast()`, otherwise fine :+1:

**Review by chrivers — CHANGES_REQUESTED** [review]

File: ankerctl.py (hunk 1 above)
Comment (billyjbryant) [reply ankerctl.py:227]: Ok, I can move it there :)

Comment (billyjbryant) [reply ankerctl.py]: ✅

**Review by chrivers — APPROVED** [review]
The whole way `AnkerPPPPApi` handles `.state` is a bit of a hack, but that's on me :)

This looks great - let's merge it :)

**Pull request merged**